vimeo/VimeoUpload
# VIM-3704: Demo project simplification

#### Ticket
**Required for Vimeans only**
[VIM-3704](https://vimean.atlassian.net/browse/VIM-3704)

#### Ticket Summary
The README should more clearly describe how to use old and new upload. Both example projects should work as expected. And the VimeoUpload object should be genericized to accommodate both old and new upload.

#### Implementation Summary
Made VimeoUpload class generic. Cleaned up and squashed some bugs in the demo projects. Updated the README.

#### How to Test
Drop and auth token into each example project and ensure that uploads are working. Peep the README and confirm that it makes sense.
